agent/agent.go

```diff
@@ -565,3 +581,29 @@ func formatReportLine(cells ...string) string {
 
 	return fmt.Sprintf(format, strValues...)
 }
+
+// GetDefaultAgentRedactions returns the default agent-level redactions that we ship with hcdiag
+func getDefaultAgentRedactions() []*redact.Redact {
+	redactions := []struct {
+		name    string
+		matcher string
+		replace string
+	}{
+		// {
```

Comment:
Yes, it's an example of how we'd add default redactions here, but I left it in a bunch of different files -- happy to clean that up if you and @mkcp prefer.

Reply:
I've removed all of these comments now.

Reply:
The repetition across files gives me a bit of pause. It seems there's a pattern here of: empty slice of anonymous structs with name, matcher, replace fields; create a slice of redact.Redact using the struct slice as input; return the slice of redact.Redact. This appears to be happening across products.

So, I'm wondering if this isn't more of a concern of the redact package. Perhaps we could use a redactConfig struct (there's probably a better name - but it would have name, matcher, and replace). We could have a function to generate a redact.Redact from such a config. And, we could have a function that would take a slice of RedactConfig and return a slice of redact.Redact. Then, each of the products would have a (currently empty) slice of RedactConfig, it could pass that to the new redact slice function, and get back what it needs. I think that would deduplicate some code and could be beneficial in the future. Plus, if we wanted to add default redactors, we could just rely on the type of `[]redact.RedactConfig` to inform us what the shape of that input needed to be.